docs/product_faq.md

```diff
@@ -0,0 +1,120 @@
+file. You compile a binary. Your binary reads input files and writes output files.
+
+Your app isn't one binary anymore. It's a managed database, and a frontend server,
+and a web app -- all talking to each other over HTTP.
+
+Development today needs new tools.
+
+[A cloud-native development environment](https://blog.tilt.dev/2019/09/05/put-down-particle-accelerator.html)
+is a new kind of tool that understands how your files and your servers fit
+together, and can help you better understand your system.
+
+### What kinds of teams should use Tilt?
+
+Currently, we're focused on helping teams that develop microservices, because that's where
+the pain is most acute.
+
+- Do you always have 5 terminal windows open to stream the logs of all your
+  servers, because
+  [you're not sure where to look](https://blog.tilt.dev/2019/04/09/designing-a-better-interface-for-microservices-development.html)?
+- Are devs [always asking questions in team chat](https://blog.tilt.dev/2019/10/01/solving-the-laggy-human-shell-problem.html) about how to distinguish meaningful error logs from noise?
+- Do you have [a complicated Bash script that sets up all your dev servers](https://blog.tilt.dev/2018/12/05/tilt-is-the-start-sh-script-of-my-dreams.html), but is always breaking?
+
+Once you've set up Tilt, any contributor should be able to run `tilt up` to get
+[a complete dev environment](https://docs.tilt.dev/welcome_to_tilt.html).
+
+### Why does Tilt focus on Kubernetes?
+
+Kubernetes defines well-thought-out building blocks for running servers together --
+such as containers, pods, and services. They're quickly becoming
+standards for our industry.
+
+We should be using these building blocks for all our development environments,
+not just when we're running on big managed clouds like AWS, AKS, or GKE.
+
+Tilt does support other systems for building and running servers. For example,
+you can run your containers with [docker-compose](docker_compose.html), or
+building with [local shell commands](api.html#api.local_resource).  But we
+expect that the industry will converge on Kubernetes. Tilt's support for other
+systems is mostly about
+[making it easier for teams to migrate](https://blog.tilt.dev/2019/09/16/tips-on-moving-your-dev-env-from-docker-compose-to-kubernetes.html).
+
+### What's Next?
+
+Tilt is just the first step towards blurring the line between your laptop and the cloud.
+
+The next step -- [Tilt Cloud](snapshots.html) -- is a platform for
+making all kinds of data from your tilt instance available to your team. And
+making your team’s data available to you.
+
+## Governance
+
+### Who Develops Tilt?
+
+We're a start-up! Call us Windmill Engineering. Nice to e-meet you.
+
+Our mission is to build a platform for cloud-native development.
+
+We’re a small team of 6 people. We’re based at
+[Work-Bench](https://www.work-bench.com/) in New York City.
+
+### If you're a startup, does that mean you will collapse?
+
+Eventually! All things must pass.
```

Comment:
Might be good to say something reassuring. "We have funding from top-tier VCs we love, and have plans, talk to us if you're nervous."?